src/test/resources/basic-validations.xml

```diff
@@ -82,6 +82,21 @@
         </validation:all>
     </flow>
 
+    <flow name="any">
+        <validation:any>
+            <validation:is-url url="#[vars.url]"/>
+            <validation:is-email email="#[vars.email]"/>
+        </validation:any>
+    </flow>
+
+    <flow name="anyWithNonValidationError">
+        <validation:any>
+            <validation:is-url url="#[vars.url]"/>
```

Comment:
add a test in which a validator from another extension is also used

Reply:
are there such tests in this module?

Reply:
that is not working, even fgor the all validator:

```
Full list:
org.xml.sax.SAXParseException; lineNumber: 42; columnNumber: 40; cvc-complex-type.2.4.a: Invalid content was found starting with element 'heisenberg:validateMoney'. One of '{"http://www.mulesoft.org/schema/mule/validation":validator}' is expected.
org.xml.sax.SAXParseException; lineNumber: 64; columnNumber: 40; cvc-complex-type.2.4.a: Invalid content was found starting with element 'heisenberg:validateMoney'. One of '{"http://www.mulesoft.org/schema/mule/validation":validator}' is expected.
 org.mule.runtime.core.api.config.ConfigurationException: There were '2' errors while parsing the given file 'aggregate-validations.xml'.```

Will raise a jira for that

Reply:
https://www.mulesoft.org/jira/browse/MULE-15778